hw-tictac/src/index.js

```diff
@@ -1,20 +1,20 @@
 import React from "react";
-import ReactDOM from "react-dom/client";
+import ReactDOM from "react-dom";
 import App from "./App";
 import { Provider } from "react-redux";
 import store from "./redux/store";
 import "./style/index.scss";
 import { CssBaseline, ThemeProvider } from "@mui/material";
 import { theme } from "alice-mui-provider";
 
-const element = document.getElementById("root");
-const root = ReactDOM.createRoot(element);
-
-root.render(
-    <Provider store={store}>
-        <ThemeProvider theme={theme}>
-            <CssBaseline enableColorScheme />
-            <App />
-        </ThemeProvider>
-    </Provider>
-);
\ No newline at end of file
+ReactDOM.render(
```

Comment:
I just removed the warning by using the latest function provided by them. Is that a problem?

Reply:
Yeah, that should be all this is -- There was a collision on the versions I believe so I just mimicked what the other applications and the application boilerplate is using in the monorepo